src/components/Navbar.tsx

```diff
@@ -0,0 +1,82 @@
+import Image from 'next/image'
+import ShoppingCart from '@mui/icons-material/ShoppingCartTwoTone'
+import Link from 'next/link'
+import AccountMenu from './AccountMenu'
+import { Badge } from '@mui/material'
+import CategoryBtn from './CategoryBtn'
+import { useState } from 'react'
+
+const Navbar = () => {
+	const [isActive, setIsActive] = useState<null | number>(null)
+
+	const mainCategories = ['WEAPONS', 'ARMOR', 'APPAREL', 'FOOD/DRINK', 'AID', 'JUNK', 'AMMO']
+	const weaponsSubCategories = ['RANGED', 'MELEE','LASER']
```

Comment:
Pls, move it outside of the component. This way, it will be created only once, not every time the component is rendered.